src/Discord.Net.Core/Entities/Interactions/Message Components/ComponentBuilder.cs

```diff
@@ -153,37 +152,31 @@ public ComponentBuilder WithSelectMenu(SelectMenuBuilder menu, int row)
         }
 
         /// <summary>
-        ///     Adds a button to the first row.
-        /// </summary>
-        /// <param name="button">The button to add to the first row.</param>
-        /// <returns>The current builder.</returns>
-        public ComponentBuilder WithButton(ButtonBuilder button)
-            => this.WithButton(button, 0);
-
-        /// <summary>
-        ///     Adds a button to the specified row.
+        ///     Adds a button to the specified row. (by default first row)
```

Comment:
remote the `(by default first row)` as its unnecessary, you will see the row = 0 parameter in the function

Reply:
Resolved in commits below